### ISSUE TITLE
Installation with pip2 fails

### ISSUE THREAD

**Issue by letorbi** [chrisjbillington/inotify_simple#16]
Hello Chris,

installing *inotify-simple* via `pip2 install enum34 inotify-simple` fails with the following error message:

```
DEPRECATION: Python 2.7 will reach the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 won't be maintained after that date. A future version of pip will drop support for Python 2.7. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting enum34
  Using cached https://files.pythonhosted.org/packages/c5/db/e56e6b4bbac7c4a06de1c50de6fe1ef3810018ae11732a50f15f62c7d050/enum34-1.1.6-py2-none-any.whl
Collecting inotify-simple
  Using cached https://files.pythonhosted.org/packages/9d/ab/c291e2e5b6d6f7575e7fdc31040dd6dbb0219fb626094184c3166bd321dc/inotify_simple-1.3.1.tar.gz
    ERROR: Command errored out with exit status 1:
     command: /usr/bin/python2 -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-EPSf8s/inotify-simple/setup.py'"'"'; __file__='"'"'/tmp/pip-install-EPSf8s/inotify-simple/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-install-EPSf8s/inotify-simple/pip-egg-info
         cwd: /tmp/pip-install-EPSf8s/inotify-simple/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-EPSf8s/inotify-simple/setup.py", line 3, in <module>
        from inotify_simple import __version__
      File "inotify_simple.py", line 3, in <module>
        from enum import Enum, IntEnum
    ImportError: No module named enum
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

This still worked with version 1.2.1. I assume the problem is the  [line 3 of setup.py](https://github.com/chrisjbillington/inotify_simple/blob/master/setup.py#L3), since you import `inotify_simple.py` here, which then tries to import `enum`, but fails. To solve the problem, I suggest to move `__version__` into an own file, which can then be imported by *setup.py* and *inotify_simple.py*.

It would also be nice to add `enum34` as a dependency if the packages is installed for Python 3.3 or lower. Maybe like this:

```
if sys.version_info.major < 3 or (sys.version_info.major == 3 and sys.version_info.minor < 4):
    install_requires = ["enum34"]

setup(name='inotify_simple',
    ...
    install_requires=install_requires,
    )
```

Regards,
Torben

**Comment by chrisjbillington** [comment]
Thanks for the report and sorry for the breakage! Fixed in 1.3.2.

I added install_requires and am `eval()`ing the version instead of importing.

**Comment by letorbi** [comment]
Wow, that was fast. Thank you :+1:
